src/db/__init__.py

```diff
@@ -43,6 +43,26 @@ def get_next_survey_question(user_id, survey_id):
     return question, answers
 
 
+def get_survey_answer_id_by_alias(alias):
```

Comment:
If you rename `find_all_by_alias` to `find_all_answer_alias_responders` then this could be `get_answer_alias_survey_answer_id`. Hm. I'm not 100% convinced about either of the names, so I think we need to discuss these namings offline.

Reply:
Ok I renamed it to get_answer_alias_survey_answer_id.
We can discuss it more if u want but